src/research-development/articles/accessibility-practices.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: layouts/article.njk
+title: "Accessibility Practices"
+date: 2022-03-30
+tags: 
+  - research_development
+  - testing
+  - open_standards
+  - techniques
+  - tools
+  - stability
+areas_of_coverage: ["Digital Service"]
+contributors: ["John Nolan"]
+---
+
+## Introduction
+
+Accessibility is one of the most important factors to be considered while building a service in Government. We build services for everyone.
+
+This article will cover the areas within the service manual and parts that aren't.
+
+We want to set out our intent while in our Alpha phase that we should hold ourselves too in the Beta phase of the service.
+
+## Web Content Accessibility Guidelines (WCAG 2.1)
+
+Reference: [Understanding WCAG 2.1](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag#wcag-design-principles)
+
+> The Web Content Accessibility Guidelines are an internationally recognised set of recommendations for improving web accessibility.
+
+Any services created will be built to the WCAG 2.1 standard.
+
+We will also ensure as newer versions of the standard get released that we meet these as part of our ongoing commitment to continuous improvement.
+
+We will do this by
+
+* Ensure all professions collaborate on early designs
+* [Adding automation to our pipelines](#automated-testing-in-pipelines)
+* Adding checks in our development environments
```

Comment:
What's the difference between adding checks and automated testing?

Reply:
More to do with built in tooling like vscode extensions and potential pre-commit hooks. Will change the wording but also tempted to just remove it, what are your thoughts, too much information and keep with automation?

Reply:
I have removed this after consideration, I think its too much and is assumed from other areas of the document